lib/rules/web/admin_console/4.2/actions.xyaml

```diff
@@ -0,0 +1,57 @@
+click_one_dropdown_action:
+  action: click_resource_action_button
+  action: choose_item_from_list
+click_resource_action_button:
+  elements:
+  - selector:
+      xpath: //button[contains(.,'Actions')]
```

Comment:
rules is changed from `/button//span[contains(text(),‘Actions’)]` to `//button[contains(.,'Actions')]`, when using `/button//span[contains(text(),‘Actions’)]` overlap is found and element is not clickable